packages/apis/src/generated/types.ts

```diff
@@ -2581,6 +2581,8 @@ export interface Resource {
     | 'annotations'
     | 'remotes'
     | 'replications'
+    | 'flows'
```

Comment:
Is this really generated? ... I guess it is not, so the next generation of APIs can yield to unpredictable results

Reply:
It is generated by our fork of `oats`, see https://github.com/bonitoo-io/oats/pull/2

Reply:
OK, .... ready to go once a solution is in `bonitoo-io/oats` master branch.